Oleksii_Fedorchuk/HW5_Oleksii_Fedorchuk/task2.py

```diff
@@ -0,0 +1,10 @@
+friends = ["John", "Marta", "James", "Amanda", "Marianna"]
+title_string = ' Names '
+print(f"{title_string}".center(50, "*"))
+for names in friends:
+    print(f"{friends[0]}".rjust(27))
+    print(f"{friends[1]}".rjust(27))
+    print(f"{friends[2]}".rjust(27))
+    print(f"{friends[3]}".rjust(27))
+    print(f"{friends[4]}".rjust(27))
+    break
```

Comment:
Em. Very strange solution. Why do you use a loop if you get value by indexes inside it?